docs/literate.jl

```diff
@@ -16,16 +16,24 @@ Pkg.instantiate()
 pkg_status = sprint() do io
     Pkg.status(; io=io)
 end
-manifest_status = sprint() do io
-    Pkg.status(; io=io, mode=Pkg.PKGMODE_MANIFEST)
-end
 
 using Literate: Literate
 
 const MANIFEST_OUT = joinpath(EXAMPLE, "Manifest.toml")
 mkpath(joinpath(OUTDIR, EXAMPLE))
 cp(joinpath(EXAMPLEPATH, "Manifest.toml"), joinpath(OUTDIR, MANIFEST_OUT); force=true)
 
+""" adapted from HttpCommon.jl """
+function escapeHTML(i::String)
```

Comment:
Does this exist in some other package? IMO it would be nice to not have to implement our own custom replacement rules which probably are incomplete and buggier than functionality in some dedicated HTML package.

Reply:
It does, as I mentioned in the docstring and explained in https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl/pull/104#discussion_r795846656

Reply:
Maybe we could just use `Markdown.htmlesc`?